source/stylesheets/scss/furatto/_tables.scss

```diff
@@ -0,0 +1,151 @@
+//Tables
+
+//
+//@variables
+//
+@import"compass";
+@import"functions";
+
+//Generalsettings
+$table-background-color: #FFF !default;
+$table-margin-bottom: px-to-rems(20) !default;
+$table-cell-padding: px-to-rems(15) px-to-rems(15) !default;
+$table-line-height: px-to-rems(18) !default;
+$table-cell-alignment: left !default;
+$table-border-color: #E8E8EB !default;
+$table-border-width: 1px !default;
+$table-border-style: solid !default;
+$table-color: #95959F !default;
+$table-font-weight: 400 !default;
+$table-font-size: px-to-rems(13) !default;
+$table-row-hover-background-color: #f9f9f9 !default;
+
+$table-head-font-weight: 600 !default;
+
+//Variations
+$table-striped-background-color: $table-row-hover-background-color !default;
+
+$table-shadow-color: rgba(0,0,0,.14) !default;
```

Comment:
`.14` should be written with a leading zero as `0.14`<br>Commas in function arguments should be followed by a single space